packages/fbjs/src/core/Deferred.js

```diff
@@ -57,7 +57,7 @@ class Deferred<Tvalue, Treason> {
   }
 
   done(): void {
-    Promise.prototype.done.apply(this._promise, arguments);
+    Promise.prototype.then.apply(this._promise, arguments);
```

Comment:
I'm not 100% sure about this change. It seems equivalent, after looking at the definition on https://www.promisejs.org/api/ (which also includes the polyfill). `done()` seems like `then()`, except it doesn't return a `Promise`, so you can't chain.

If I'm wrong, I'll just change this to 

    (Promise.prototype.done: any).apply(this._promise, arguments);

but I thought I'd rather try this first.